homeassistant/components/mysensors/manifest.json

```diff
@@ -3,7 +3,8 @@
   "name": "MySensors",
   "documentation": "https://www.home-assistant.io/integrations/mysensors",
   "requirements": [
-    "pymysensors==0.20.1"
+    "pymysensors==0.20.1",
+    "packaging==20.8"
```

Comment:
I don't think we need this since we require `setuptools` to install home assistant and `setuptools` requires `packaging`.

Reply:
It was inconsistent for me. In one of my venvs it was there already but in the other it was missing. After i have added the requirement it worked.
But if you think it isn't needed then i will remove it.

Reply:
`setuptools` has vendored `packaging` for three years. If the venv was created longer than three years ago it may not be included.

We can keep it for now but probably we'll switch to `awesomeversion` instead. If that becomes a core requirement we don't need a custom requirement here.

Reply:
Please add `packaging` back to the manifest for now. We can clean it up later when we switch to `awesomeversion`.

Reply:
We just added `awesomeversion` as a core requirement in Home Assistant. So now we can use that instead. Please change the code to use that package and remove the requirement on `packaging`here.

https://pypi.org/project/awesomeversion/